### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -21513,6 +21522,8 @@ persuing->pursuing
 persuit->pursuit
 persuits->pursuits
 persumably->presumably
+perticular->particular
+perticularly->particularly
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:21526]: How about adding `perticulars->particulars`
https://github.com/search?q=perticulars&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -22421,7 +22432,9 @@ prinicipal->principal
 printting->printing
 prioritiy->priority
 priorty->priority
+priot->prior
 priotity->priority
+priots->priors
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:22437]: How about adding `priotities->priorities`
https://github.com/search?q=priotities&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:22445]: vikivivi

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:21526]: ff937de

**Comment by vikivivi** [comment]
LGTM

**Pull request merged**